realm/realm-library/src/androidTestObjectServer/java/io/realm/SyncManagerTests.java

```diff
@@ -153,14 +139,14 @@ public void loggedOut(SyncUser user) {
         assertEquals(0, counter[0]);
         assertEquals(0, counter[1]);
     }
-
     @Test
-    public void session() {
+    public void session() throws IOException {
         SyncUser user = createTestUser();
         String url = "realm://objectserver.realm.io/default";
         SyncConfiguration config = new SyncConfiguration.Builder(user, url)
                 .build();
-
+        // This will trigger the creation of the session
```

Comment:
The ObjectStore creates automatically the session when you first open a Realm(SharedRealm) 

`SyncManager.getSession(config)` does only create the Java wrapper (if it doesn't exist) around the previously ObjectStore session.

Hence the need to call `Realm.getInstance(config);`